julia.html.markdown

```diff
@@ -765,9 +765,9 @@ code_native(square_area, (Int32,), syntax = :intel)
 	#         nop     dword ptr [rax + rax]
 	# ;}
 
-code_native(square_area, (Float32,), syntax = :intel)
+codeNative(squareArea, (Float32,), syntax = :intel)
     #         .text
-    # ; Function square_area {
+    # ; Function squareArea {
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194